docs/USAGE.md

```diff
@@ -2,37 +2,95 @@
+
+It is recommended to use OVE with Google Chrome, as this is the web browser used for development and in production at the [Data Science Institute](http://www.imperial.ac.uk/data-science/). However, it should also be compatible with other modern web browsers: if you encounter any browser-specific bugs please [report them as an Issue](https://github.com/ove/ove-apps/issues).
+
+To test OVE functionality, you can simply open four web browser windows (or tabs) with the following URLs to create the `LocalFour` space:
+
+```text
+http://OVE_CORE_HOST:PORT/view.html?oveViewId=LocalFour-0
+http://OVE_CORE_HOST:PORT/view.html?oveViewId=LocalFour-1
+http://OVE_CORE_HOST:PORT/view.html?oveViewId=LocalFour-2
+http://OVE_CORE_HOST:PORT/view.html?oveViewId=LocalFour-3
+```
+
+However, in a much larger OVE installation with many monitors, opening browser windows may be sufficiently time-consuming for automation to be worthwhile.
+
+In a Windows environment, a [PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/powershell-scripting) script can be used to launch full-screen browsers with the correct URLs. In a Linux or MacOS environment, a [Bash](https://linux.die.net/man/1/bash) script can be used to launch full-screen browsers with the correct URLs. In a MacOS environment, [AppleScript](https://developer.apple.com/library/archive/documentation/AppleScript/Conceptual/AppleScriptLangGuide/introduction/ASLR_intro.html) can be used as an alternative to [Bash](https://linux.die.net/man/1/bash).
+
+## Launching OVE Apps
+
+OVE provides three different ways of launching apps into an OVE environment:
+
+* Using [**OVE Editor**](https://github.com/ove/ove-editor)
+* Using [**OVE SDKs**](https://github.com/ove/ove-sdks)
+* Using [**OVE Core**](https://github.com/ove/ove) APIs
+
+The [**OVE Editor**](https://github.com/ove/ove-editor) is designed to be used by the most basic users of OVE. The [**OVE Core**](https://github.com/ove/ove) APIs are intended to be used by the most advanced users of OVE.
+
+Given below are instructions on how to load sample content using the [Image App](https://github.com/ove/ove-apps/tree/master/packages/ove-app-images). A complete list of all apps and similar instructions on how to use them can be found in the [**OVE Apps**](https://github.com/ove/ove-apps) repository.
+
+### Launching OVE Apps from the OVE Editor
+
+The [**OVE Editor**](https://github.com/ove/ove-editor) is still work in progress, and is not currently available as an option.
+
+### Launching OVE Apps using the Python Client Library
+
+The [Python Client Library](https://github.com/ove/ove-sdks/tree/master/python) (installation instructions available on GitHub repository) is one of the SDKs provided by OVE. To launch the [Image App](https://github.com/ove/ove-apps/tree/master/packages/ove-app-images) with a sample image, in the `LocalNine` space, run:
```

Comment:
I don't think that "(installation instructions available on GitHub repository)" is necessary: you see the instructions as soon as you follow the link.

Reply:
We have had situations where people did not realise they had to click the link to figure that out. I will therefore leave this as-is.

Reply:
Ok - it would perhaps be more elegant to write "The [Python Client Library](https://github.com/ove/ove-sdks/tree/master/python) ([installation instructions](https://github.com/ove/ove-sdks/blob/master/python/README.md))"

i.e.,

     "The [Python Client Library](https://github.com/ove/ove-sdks/tree/master/python) ([installation instructions](https://github.com/ove/ove-sdks/blob/master/python/README.md))"